web/public/index.html

```
@@ -0,0 +1,147 @@
+    </section>
+
+    <section id="tab-picks" class="tab">
+      <div class="toolbar">
+        <label>Score min
+          <select id="picks-min-score">
+            <option value="3.4" selected>≥ 3.4 (APPLY)</option>
+            <option value="3.0">≥ 3.0 (incl. conditional)</option>
+            <option value="2.5">≥ 2.5 (incl. borderline)</option>
+          </select>
+        </label>
+        <input id="picks-search" type="search" placeholder="Filtrer (entreprise, rôle)" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)